src/Mobs/Cow.h

```diff
@@ -20,7 +20,10 @@ class cCow :
 	virtual void GetDrops(cItems & a_Drops, cEntity * a_Killer = nullptr) override;
 	virtual void OnRightClicked(cPlayer & a_Player) override;
 
-	virtual const cItem GetFollowedItem(void) const override { return cItem(E_ITEM_WHEAT); }
+	virtual void GetFollowedItems(cItems & a_Items) override
```

Comment:
Now thats a better reason. And my answer is that ConvertToPickups should also work like this, but because the signature predates the adoption of c++11 in this project, it made sense to do it by reference at the time.